python/docs/source/development/testing.rst

```diff
@@ -36,6 +36,13 @@ After that, the PySpark test cases can be run via using ``python/run-tests``. Fo
 
     python/run-tests --python-executable=python3
 
+You can run individual tests by using ``--testnames`` option. For example,
+
+.. code-block:: bash
+
+    python/run-tests --testnames pyspark.sql.tests.test_dataframe
+    python/run-tests --testnames pyspark.sql.tests.test_dataframe.DataFrameTests.test_range
+
```

Comment:
Shall we add version supported? If users are on older Spark versions this won't work right?